### PR TITLE
Improve Readme & add Emacs/ASDF support to the repo

### PR DESCRIPTION
**Pull request by m-doughty**
This pull request:

- Improves the clarity of the readme;
- Adds a dotfile to support Yarn PnP in Emacs;
- Adds a dotfile to correct the nodejs version with asdf.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,11 +55,11 @@ For you, a newcomer, to be running your development setup, you'll need to comple
 
 - Log in to <https://cloud.squidex.io/app/asap-hub-dev>
 - In `Content/Users`, create a new user (or edit if already created)
-- While creating, or editing the user, on `Functional/Connections`, add a new item to store your UUID (you can generate a random one [here](https://www.uuidgenerator.net/version4)) and save that (the save button is the blue one on top!). You'll need that UUID later, so save it.
+- While creating, or editing the user, on `Technical/Connections`, add a new item to store your UUID (you can generate a random one [here](https://www.uuidgenerator.net/version4)) and save that (the save button is the blue one on top!). You'll need that UUID later, so save it.
```
Comment (kmaid) [inline README.md:58]: This isn't true anymore. When you save a user a UUID is generated when you get the result back

File: README.md [hunk 2]
```diff
@@ -45,11 +55,11 @@ For you, a newcomer, to be running your development setup, you'll need to comple
...
 
 ### Get everything running
 
-- create a `.env` file and update it with the necessary details (ask someone for this)
+- create a `.env` file and update it with the necessary details (ask one of the engineers for a working config)
```
Comment (kmaid) [inline README.md:62]: Maybe point out there is a .env.example to start with

**Review by kmaid — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (m-doughty) [reply README.md:58]: Thanks, I'll remove this

**Pull request merged**